.gitignore

```diff
@@ -34,6 +34,7 @@ tree.txt
 test.ipynb
 test.json
 test.py
+TopicMemory.json
```

Comment:
Good point, that's not even supposed to be there. You can ignore that file here for now, but I'll move it / take care of it at some point.